src/Microsoft.ML.Data/Commands/CrossValidationCommand.cs

```diff
@@ -57,8 +57,9 @@ public sealed class Arguments : DataCommand.ArgumentsBase
             [Argument(ArgumentType.LastOccurenceWins, HelpText = "Column to use for stratification", ShortName = "strat", SortOrder = 7)]
             public string StratificationColumn;
 
-            [Argument(ArgumentType.LastOccurenceWins, HelpText = "Columns with custom kinds declared through key assignments, for example, col[Kind]=Name to assign column named 'Name' kind 'Kind'", ShortName = "col", SortOrder = 10)]
-            public KeyValuePair<string, string>[] CustomColumn;
+            [Argument(ArgumentType.LastOccurenceWins, HelpText = "Columns with custom kinds declared through key assignments, for example, col[Kind]=Name to assign column named 'Name' kind 'Kind'",
+                Name = "CustomColumn", ShortName = "col", SortOrder = 10)]
```

Comment:
>CustomColumn [](start = 24, length = 12)

So should this be 'CustomColumns' ?  

(here, and everywhere else)

Reply:
No. This is so that the behavior remains the same for the command line. Similarly for 'Column'. See #2040 for details.

Reply:
Ah ok. Thanks for the clarification.

---
In reply to: [247224979](https://github.com/dotnet/machinelearning/pull/2124#discussion_r247224979) [](ancestors = 247224979)